oak_tensorflow_service/src/tflite.rs

```diff
@@ -33,10 +33,11 @@ extern "C" {
 
     /// Performs inference on `input_bytes` and pass inference result to `output_bytes` and returns
     /// an error code.
+    /// `output_bytes` are allocated and freed by TensorFlow Lite.
```

Comment:
What does this mean? As the caller of this function, do I need to worry about allocation or not?